# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1340288319 -7200
#      Thu Jun 21 16:18:39 2012 +0200
# Branch gui
# Node ID 7e41fcbb23a4cdf10a873d13e5dc86fca8280b73
# Parent  84d1d9c035c70d028186bbdd190dda637ca0969c
Fixed shortcuts in file editor.

* file-editor.cc: Changed shortcut context.

diff --git a/gui/src/m-editor/file-editor.cc b/gui/src/m-editor/file-editor.cc
--- a/gui/src/m-editor/file-editor.cc
+++ b/gui/src/m-editor/file-editor.cc
@@ -331,45 +331,45 @@ file_editor::construct ()
         tr("&Run File"), _tool_bar);
 
   // some actions are disabled from the beginning
   _copy_action->setEnabled(false);
   _cut_action->setEnabled(false);
 
   // short cuts
   new_action->setShortcut                       (QKeySequence::New);
-  new_action->setShortcutContext                (Qt::WidgetShortcut);
+  new_action->setShortcutContext                (Qt::WindowShortcut);
   open_action->setShortcut                      (QKeySequence::Open);
-  open_action->setShortcutContext               (Qt::WidgetShortcut);
+  open_action->setShortcutContext               (Qt::WindowShortcut);
   save_action->setShortcut                      (QKeySequence::Save);
-  save_action->setShortcutContext               (Qt::WidgetShortcut);
+  save_action->setShortcutContext               (Qt::WindowShortcut);
   save_as_action->setShortcut                   (QKeySequence::SaveAs);
-  save_as_action->setShortcutContext            (Qt::WidgetShortcut);
+  save_as_action->setShortcutContext            (Qt::WindowShortcut);
   undo_action->setShortcut                      (QKeySequence::Undo);
-  undo_action->setShortcutContext               (Qt::WidgetShortcut);
+  undo_action->setShortcutContext               (Qt::WindowShortcut);
   redo_action->setShortcut                      (QKeySequence::Redo);
-  redo_action->setShortcutContext               (Qt::WidgetShortcut);
+  redo_action->setShortcutContext               (Qt::WindowShortcut);
   _copy_action->setShortcut                     (QKeySequence::Copy);
-  _copy_action->setShortcutContext              (Qt::WidgetShortcut);
+  _copy_action->setShortcutContext              (Qt::WindowShortcut);
   _cut_action->setShortcut                      (QKeySequence::Cut);
-  _cut_action->setShortcutContext               (Qt::WidgetShortcut);
+  _cut_action->setShortcutContext               (Qt::WindowShortcut);
   paste_action->setShortcut                     (QKeySequence::Paste);
-  paste_action->setShortcutContext              (Qt::WidgetShortcut);
+  paste_action->setShortcutContext              (Qt::WindowShortcut);
   run_action->setShortcut                       (Qt::Key_F5);
-  run_action->setShortcutContext                (Qt::WidgetShortcut);
+  run_action->setShortcutContext                (Qt::WindowShortcut);
   next_bookmark_action->setShortcut             (Qt::Key_F2);
-  next_bookmark_action->setShortcutContext      (Qt::WidgetShortcut);
+  next_bookmark_action->setShortcutContext      (Qt::WindowShortcut);
   previous_bookmark_action->setShortcut         (Qt::SHIFT + Qt::Key_F2);
-  previous_bookmark_action->setShortcutContext  (Qt::WidgetShortcut);
+  previous_bookmark_action->setShortcutContext  (Qt::WindowShortcut);
   toggle_bookmark_action->setShortcut           (Qt::Key_F7);
-  toggle_bookmark_action->setShortcutContext    (Qt::WidgetShortcut);
+  toggle_bookmark_action->setShortcutContext    (Qt::WindowShortcut);
   comment_selection_action->setShortcut         (Qt::CTRL + Qt::Key_R);
-  comment_selection_action->setShortcutContext  (Qt::WidgetShortcut);
+  comment_selection_action->setShortcutContext  (Qt::WindowShortcut);
   uncomment_selection_action->setShortcut       (Qt::CTRL + Qt::Key_T);
-  uncomment_selection_action->setShortcutContext(Qt::WidgetShortcut);
+  uncomment_selection_action->setShortcutContext(Qt::WindowShortcut);
 
   // toolbar
   _tool_bar->addAction (new_action);
   _tool_bar->addAction (open_action);
   _tool_bar->addAction (save_action);
   _tool_bar->addAction (save_as_action);
   _tool_bar->addSeparator ();
   _tool_bar->addAction (undo_action);
